qemu/tests/vpmu_check_instructions.py

```diff
@@ -0,0 +1,63 @@
+import os
+import logging
+import re
+import platform
+
+import aexpect
+from virttest import data_dir
+from virttest import error_context
+from avocado.utils import process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    vpum cpu cycles checking between host and guest:
+    1) boot guest
+    2) check cpu cycles for host
+    3) check cpu cycles for guest and compare wirh host
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    check_timeout = int(params.get("check_timeout", "600"))
+    tmp_dir = params.get("tmp_dir", "/root")
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+    test_cmd = params.get("test_cmd")
+    build_cmd = params.get("build_cmd")
+    vm_arch = params["vm_arch_name"]
+    host_arch = platform.machine()
+    src_dir = os.path.join(data_dir.get_deps_dir(), 'million')
+    src_file = os.path.join(src_dir, "million-%s.s" % host_arch)
+    dst_file = os.path.join(tmp_dir, "million-%s.s" % host_arch)
+    if process.system("test -x perf", ignore_status=True, shell=True) != 0:
+        process.getstatusoutput("yum install -y perf")
+
+    error_context.context("build binary file 'million' in host", logging.info)
+    os.chdir(src_dir)
+    process.getstatusoutput(build_cmd % host_arch)
+
+    error_context.context("running test command in host", logging.info)
+    s, o = process.getstatusoutput(test_cmd)
```

Comment:
it seems, it returned "status" but you didn't check it

Reply:
A reference for L40 to L45
cmds.append()
for cmd in cmds:
s, o = process.getstatusoutput(cmd)
if s:
test.error/fail("")